### PR TITLE
Round function

### PR DESCRIPTION
**Pull request by Lucs1590**
Function used to rounds a number.

### CODE REVIEW

**Comment by Lucs1590** [comment]
I think the code climate will not work that way. They have changed some things and it is necessary to add something of this type:


      before_script:
        - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
        - chmod +x ./cc-test-reporter
        - ./cc-test-reporter before-build

      after_script:
        - ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT

File: src/object-utils.ts [hunk 1]
```diff
@@ -207,5 +207,24 @@ export class ObjectUtils {
       }
       return formatedMessage;
     }
-  }
+
+    /**
+     * Function used to rounds a number
+     * Exemple 1:
```
Comment (rafaelim) [inline src/object-utils.ts]: You could use @example to make a better doc.

**Review by ettoreleandrotognoli — APPROVED** [review]

**Comment by ettoreleandrotognoli** [comment]
if i'm not wrong the code climate should works because of this: 
https://github.com/OpenCIAg/ts-commons/blob/28b4e1180f9e15c1ff4d76c0fdc427ab053c2a11/package.json#L23

File: src/object-utils.ts (hunk 1 above)
Comment (Lucs1590) [reply src/object-utils.ts]: Nice, then I'lll change it. I never did that, but I liked the idea.

Comment (Lucs1590) [reply src/object-utils.ts]: I followed this documentation:
https://devdocs.io/jsdoc/tags-example#overview

**Review by rafaelim — APPROVED** [review]

**Pull request merged**